scss/_callout.scss

```
@@ -31,15 +31,19 @@
   }
 }
 
-.callout-dark {
-  @extend .bg-dark;
+@each $color, $value in $theme-colors {
+  .callout-#{$color} {
+    @include callout-variant($value);
+  }
 }
 
 // ensure color contrast
 .callout-silver,
 .callout-cool-gray,
 .callout-warm-gray,
-.callout-sky {
+.callout-sky,
+.callout-info,
+.callout-light {
```

Lighthouse complained about the color contrast for leaf, success, and warning so adding those here.
```suggestion
.callout-leaf,
.callout-sky,
.callout-info,
.callout-light,
.callout-success,
.callout-warning {
```